packages/flutter/test/widget/animated_size_test.dart

```diff
@@ -37,6 +38,7 @@ void main() {
       new Center(
         child: new AnimatedSize(
           duration: new Duration(milliseconds: 200),
+          tickerProvider: TickerProvider.schedulerTickerProvider,
```

Comment:
This was an excellent idea. Done.

As a side-effect it also allows the tester to verify that all these AnimationControllers are disposed as well.